Common/UnitDefinitions/ElectricImpedance.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Name": "ElectricImpedance",
+  "BaseUnit": "Ohm",
+  "XmlDocSummary": "Electric impedance is the opposition to alternating current presented by the combined effect of resistance and reactance in a circuit. It is defined as the inverse of admittance. The SI unit of impedance is the ohm (symbol Ω).",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Electrical_impedance",
+  "ObsoleteText": "Impedance is a complex number, which is not currently supported by UnitsNet. Please use either ElectricResistance or ElectricReactance instead.",
```

Comment:
Should we still add this, if it's being obsoleted out of the box?

Reply:
ElectricAdmittance and ElectricImpedance are two sides of the same coin. See my comments for ElectricAdmittance.json for more details.

Reply:
Alright, I guess it can help stave off future questions about why it isn't added.